api/v1/ytsaurus_types.go

```diff
@@ -618,10 +622,33 @@ type TabletCellBundleInfo struct {
 	TabletCellCount int    `yson:"tablet_cell_count,attr" json:"tabletCellCount"`
 }
 
+type UpdateSelector string
+
+const (
+	UpdateSelectorNone            UpdateSelector = ""
```

Comment:
"None" is confusing, especially when it is not "Nothing".
Maybe "Unspecified".

Reply:
done